### ISSUE TITLE
Broken example in Chapter 8 - "Working with Alternatives"

### ISSUE THREAD

**Issue by Liance** [videlais/Unofficial-Ink-Cookbook#2]
Hi! There appears to be a broken example in [Chapter 8 - "Working with Alternatives"](https://github.com/videlais/Unofficial-Ink-Cookbook/blob/master/Chapter8/index.md#working-with-alternatives).

The following example will only work

```
VAR day = ""

~ day = {~Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Sunday}
```

if the alternative is in fact encapsulated by quotation marks like so

```
VAR day = ""

~ day = "{~Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Sunday}"
```
Also, it's probably worth mentioning that this pattern may not work as expected according to [the official docs](https://github.com/inkle/ink/blob/master/Documentation/WritingWithInk.md#evaluating-strings) - it may be worth noting that the shuffle's result will be locked in until it gets rolled again.

So 

```
VAR day = ""

~ day = "{~Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Sunday}"

{day} and {day} are both days of the week.
```
will produce "Wednesday and Wednesday are both days of the week.", where the days are locked. I'd suggest linking to the random day example in [this handbook, here](https://github.com/videlais/Unofficial-Ink-Cookbook/blob/master/Chapter9/index.md#ink-functions), which solves this issue.

**Comment by videlais** [comment]
Thanks for the catch, @Liance!

I agree with you about the quotation marks. I've now fixed that.

I've also added a sentence after the example clarifying that the value of the variable is not the shuffle itself, but one of its elements. That should work towards the point you make about using the variable a second time and potentially expecting a different value.

 I don't think linking to an example that uses a function is a good idea in this chapter. That concept appears in the next chapter more in-depth.
